cmd/tailscale/ui.go

```diff
@@ -839,15 +838,25 @@ func (ui *UI) layoutPeer(gtx layout.Context, sysIns system.Insets, p *UIPeer, us
 					})
 				}),
 				layout.Rigid(func(gtx C) D {
-					var addrs []string
+					var addr string
```

Comment:
I"d move this down closer to its use.

But simpler and fewer string allocations might be:

```go
    var bestIP netaddr.IP // IP to show; first IPv4, or first IPv6 if no IPv4
    for _, addr := range p.Peer.Addresses {
           if ip := addr.IP(); bestIP.IsZero() || bestIP.Is6() && ip.Is4() {
                 bestIP = ip
           }
     }
     l := material.Body2(ui.theme, bestIP.String())

Reply:
Done.